lib/couchbase/search_options.rb

```diff
@@ -53,17 +63,16 @@ def initialize(match)
           yield self if block_given?
         end
 
-        # @return [String]
-        def to_json(*args)
-          data = {"match" => @match}
-          data["boost"] = boost if boost
-          data["field"] = field if field
-          data["analyzer"] = analyzer if analyzer
-          if fuzziness
-            data["fuzziness"] = fuzziness
-            data["prefix_length"] = prefix_length if prefix_length
-          end
-          data.to_json(*args)
+        # @return [Hash<Symbol, #to_json>]
+        def to_h
+          data = {:match => @match}
+          data[:boost] = boost if boost
+          data[:field] = field if field
+          data[:analyzer] = analyzer if analyzer
+          data[:operator] = operator if operator
+          data[:fuzziness] = fuzziness if fuzziness
+          data[:prefix_length] = prefix_length if prefix_length
```

Comment:
Nervermind, the spec does not say it. I don't know why I did it initially

Reply:
Probably because it is not used with zero fuzziness:

https://github.com/blevesearch/bleve/blob/28ee4f116e20b18043437f4d00c9130c80eb5558/search/query/match.go#L141-L150

Reply:
Interesting, I guess in that case it doesn't really make a difference